lighthouse/helm/templates/validator-deployment.yaml

```diff
@@ -0,0 +1,59 @@
+{{- range $key, $validatorClient := .Values.validatorClients }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ $validatorClient.name }}
+  namespace: {{ $.Values.namespace }}
+  labels:
+    app: {{ $validatorClient.name }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ $validatorClient.name }}
+  replicas: 1
+  strategy: 
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: {{ $validatorClient.name }}
+    spec:
+      initContainers:
+      - name: init-wait
+        image: alpine:3.13.2
+        command: ['sh', '-c', 'echo "Wait for {{ $.Values.validatorStartWaitTime }} second(s) for extra slashing protection!" && sleep {{ $.Values.validatorStartWaitTime }}']
+      containers:
+      - name: {{ $validatorClient.name }}
+        image: "{{ $.Values.image.validatorImage }}:{{ $.Values.image.version }}"
+        args:
+        - lighthouse
+        {{- if $.Values.ethereumTestnet }}
+        - --network={{ $.Values.ethereumTestnet }}
+        {{- end }}
+        - vc
+        - --beacon-nodes=http://{{ $.Values.beacon.name }}-service.{{ $.Values.namespace }}.svc.cluster.local:5052
+        - --datadir={{ $validatorClient.dataVolumePath }}
```

Comment:
For the helm charts for other clients, this field is usually fixed. The reason this is a variable set as the same as dataVolumePath is because lighthouse seems to look for the voting_keystore_path in validator_definitions.yaml to decide the location of validator keys. In our case, the voting_keystore_path contains dataVolumePath instead of the fixed value we usually set in the helm chart.

Reply:
nice one!